<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Unit tests" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.004665">
    <testsuite name="SampleTest" file="/var/www/html/piedranker/test/SampleTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.004665">
      <testcase name="testTomething" class="SampleTest" classname="SampleTest" file="/var/www/html/piedranker/test/SampleTest.php" line="8" assertions="0" time="0.004665">
        <error type="PHPUnit\Framework\RiskyTestError">Risky Test
</error>
        <system-out>string(321) "{"74":{"mode":"Fastest Mode","language":"Python","number_of_questions":1,"default_score":0,"ordering":0,"name":"Assemly Lab1","duedate":1566601200,"fullname":"aaa"},"72":{"mode":"Fastest Mode","language":"Python","number_of_questions":1,"default_score":0,"ordering":0,"name":"fukl","duedate":1566255600,"fullname":"aaa"}}"
</system-out>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
